analysers/analyser_merge_shop_FR.py

```diff
@@ -62,18 +65,18 @@ def __init__(self, config, error_file, logger, classs, title, selectTags, genera
                 select = Select(
                     types = ['nodes', 'ways'],
                     tags = selectTags),
-                osmRef = "ref:FR:SIRET",
+                #osmRef = "ref:FR:SIRET",
```

Comment:
Please rebase. It's was already disabled.

Reply:
ok